docs/source/essentials/server.md

```diff
@@ -129,24 +65,24 @@ server.listen().then(({ url }) => {
 
 > By default, Apollo Server listens on port 4000.  See the [API reference](../api/apollo-server.html) for additional `listen` options, including how to configure the port.
 
-### Run the application
+<h3 id="running">Running the server</h3>
 
 With the above configuration complete, we can now start the Node application, with Apollo Server, for the first time.  This varies, but assuming a standard `index.js` configuration, might be as simple as `node index.js`.
 
 After you start the server it should print a message to the console indicating that it's ready:
 
 ```shell
 $ node index.js
-🚀 Server ready at http://localhost:4000/graphiql
+🚀 Server ready at http://localhost:4000/
 ```
 
 At this point, if the message isn't printed to the console, it's possible that something went wrong.  Double-check the previous steps in this guide, and try comparing the configuration to our [pre-configured example on Glitch]().
 
-## Explore
+<h3 id="running">GraphQL Playground</h3>
 
-To explore the newly created GraphQL server, open a browser to the link shown in the console, http://localhost:4000/graphiql.
+To explore the newly created GraphQL server, open a browser to the link shown in the console, http://localhost:4000/graphql. Apollo Server sets up GraphQL Playground for you so you can immediately start running queries and exploring your schema.
```

Comment:
http://localhost:4000/graphql => code quotes please 😄

Reply:
> Apollo Server sets up GraphQL Playground for you so you can immediately start running queries.

=> Apollo Server sets up GraphQL Playground for you so that you can start running queries and exploring schemas quickly.